src/mappings/utils.ts

```diff
@@ -122,3 +129,28 @@ export async function trackUnprocessed(error: Error, primitives: Primitives): Pr
     logger.error(`[trackUnprocessable] (ERROR | stack): ${error.stack}`);
   }
 }
+
+export function getBalanceOfAccountByDenomId(address: string, denom: string): string {
```

Comment:
Same suggestion here, if you rename `BalanceOfAccountByDenom`, I think this would become `getBalanceId()`, which is much more readable IMHO.